lib/puppet/provider/chocolateysource/windows.rb

```diff
@@ -228,7 +229,7 @@ def flush
 
     if property_ensure == :present
       begin
-        Puppet::Util::Execution.execute([command(:chocolatey), 'source', 'enable', '--name', resource[:name]])
+        Puppet::Util::Execution.execute([command(:chocolatey), 'source', 'enable', '--name', resource[:name]], sensitive: true)
```

Comment:
Would it make sense to use *args and opts here too?

Reply:
Not sure, think it works the way it is since it's the same command every time, there's no variation.

Reply:
I had thought that the args array might been useful here.. but i see that this is a second command so think its fine as is.